.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
```

Comment:
I turned [this rule](https://eslint.org/docs/rules/no-bitwise) off because I think it's nonsensical. There are lots of things that can only be done using bitwise operators, and they're an intrinsic part of JavaScript. The only justification ESLint gives is that people might accidentally use `|` instead of `||` or `&` instead of `&&`, and I don't think that's a particularly significant concern.